testsuite/tests/shadow_include/shadow_all.ml

```diff
@@ -1,5 +1,6 @@
 (* TEST
    * expect
+   flags = "-nopervasives" (* can't pass -nostdlib because of objects. *)
```

Comment:
I was a bit worried about this part, but this file does not seem to cause issues in the various stdlib improvement PRs.

Reply:
Yes, I think the idents will shift if someone ever changes the content of `CamlinternalObj`. But I think I can live with that.